README.rst

```diff
@@ -640,8 +668,8 @@ NOTES:
 1. edxapp and IDAs use the ``latest`` tag for configuration changes which have been merged to master branch of
    their repository and ``edx/configuration``.
 2. Images for a named Open edX release are built from the corresponding branch
```

Comment:
from now on we're editing upstream content, so I'd rather to keep changes to the minimal.